### ISSUE TITLE
rebar3 edoc fail

### ISSUE THREAD

**Issue by benoitc** [helium/erlang-libp2p#152]
edocs fail with the following error:

```shell
/Users/benoitc/BarrelProject/erlang-libp2p/src/peerbook/libp2p_peer.erl, function is_similar/2: at line 227: `-quote ended unexpectedly at line 229
edoc: skipping source file '/Users/benoitc/BarrelProject/erlang-libp2p/src/peerbook/libp2p_peer.erl': {'EXIT',
                                                                                                 error}.
edoc: error in doclet 'edoc_doclet': {'EXIT',error}.
===> Failed to generate documentation for app 'libp2p'
```

**Comment by tsloughter** [comment]
Maybe needs `{preprocess, true}` in edoc_opts?

**Comment by macpie** [comment]
Should be fixed with #201
